app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list[Person]:
+    Person.people = {person_info["name"]: Person(person_info["name"],
+                                                 person_info["age"])
+                     for person_info in people}
```

Comment:
There is no need to perform dict comprehension and assign values to `Person.people`. Make a list comprehension with the creation of instances which will automatically populate `Person.people` since this operation is done in the `__init__`
```suggestion
    [Person(person_info["name"], person_info["age"])
     for person_info in people]
```